Makefile

```diff
@@ -51,4 +51,9 @@ relay-gossip:
 	go build -o drand-relay-gossip -mod=readonly -ldflags "-X github.com/drand/drand/cmd/relay-gossip.version=`git describe --tags` -X github.com/drand/drand/cmd/relay-gossip.buildDate=`date -u +%d/%m/%Y@%H:%M:%S` -X github.com/drand/drand/cmd/relay-gossip.gitCommit=`git rev-parse HEAD`" ./cmd/relay-gossip
 drand-relay-gossip: relay-gossip
 
-.PHONY: client drand relay-http relay-gossip
+# create the "drand-relay-s3" binary in the current folder
+relay-s3:
+	go build -o drand-relay-s3 -mod=readonly -ldflags "-X github.com/drand/drand/cmd/relay-s3.version=`git describe --tags` -X github.com/drand/drand/cmd/relay-s3.buildDate=`date -u +%d/%m/%Y@%H:%M:%S` -X github.com/drand/drand/cmd/relay-s3.gitCommit=`git rev-parse HEAD`" ./cmd/relay-s3
+drand-relay-s3: relay-s3
+
+.PHONY: client drand relay-http relay-gossip relay-s3
```

Comment:
can you remove this bottom .PHONY line since it's actually i think ignored as a re-declaration of the 1st line in this file